app/code/Magento/CatalogGraphQl/etc/schema.graphqls

```diff
@@ -10,6 +10,10 @@ type Query {
         sort: ProductAttributeSortInput @doc(description: "Specifies which attributes to sort on, and whether to return the results in ascending or descending order.")
     ): Products
     @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Products") @doc(description: "The products query searches for products that match the criteria specified in the search and filter attributes.") @cache(cacheIdentity: "Magento\\CatalogGraphQl\\Model\\Resolver\\Product\\Identity")
+    product (
+        id: Int @doc(description: "Id of the product.")
+    ): ProductInterface
+    @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\ProductId") @doc(description: "The product query searches for product that match the criteria specified in the search and filter attributes.") @cache(cacheIdentity: "Magento\\CatalogGraphQl\\Model\\Resolver\\Product\\ProductIdIdentity")
```

Comment:
"that match the criteria specified in the search and filter attributes" doesn't seem correct if the query only has an attribute to specify the id of the product.

"The product query searches for a product that matches the specified id." feels a bit more closer to what it should be, looking at the attribute.

Reply:
Thank you for your suggestion, the description has been adjusted.